app/src/forms/form/service.js

```diff
@@ -722,6 +724,30 @@ const service = {
     }
   },
 
+  // toggle filesApiAccess for the api key
```

Comment:
I'm wondering if instead of toggling the value, that we might want to pass in the desired true/false value? What I'm wondering is that if I want to set it to false, I have to first get the value to make sure that it is true, and then call this - but if I could just pass in "false" as the desired value, then it will be false even if it was false before?

What do you think? I don't think it's a big deal since all the API Key stuff is internal to CHEFS and will probably never be made available to basic auth users. Just a thought!

Reply:
I'm a little confused. Were you thinking in terms of limiting the API calls? If so, you are correct, I can reduce the call in this function by passing in the variable from the frontend call. But since I have set this up as a toggle, there will never be a case where "false" will be passed if the value is already false. 

Also, isn't the API key already being used for basic auth users?

Reply:
Sorry for the confusion - what I'm thinking is that we want the API call to set a value (true or false), not toggle the value (change whatever it might be). For example, if I don't know what the value is now but I want to set it to false, I'd have to do (pseudo code):
```
state = GET filesApiAccess
if (state == true) {
 PUT filesApiAccess
}
```
where what I want to do is just:
```
PUT filesApiAccess false
```
I think it will be a lot simpler down the road if we set an explicit true/false value instead of toggling the existing value.

Reply:
Hey Walter, I have made the changes you requested. I am now sending the value of true/false from the frontend and explicitly setting it. Let me know if this works.

Reply:
Looks great!